haaska.py

```diff
@@ -364,6 +402,15 @@ def set_percentage(self, val):
         brightness = (val / 100.0) * 255.0
         self._call_service('light/turn_on', {'brightness': brightness})
 
+    def set_color(self, hue, saturation, brightness):
+        red, green, blue = colorsys.hsv_to_rgb(hue, saturation, brightness)
+        rgb = [int(red), int(green), int(blue)]
```

Comment:
colorsys's documentation indicates that the returned values are between 0.0 and 1.0, but doesn't Home Assistant expect rgb_color to be a 3-tuple of integers between 0 and 255?